evaluation.py

```diff
@@ -134,7 +135,7 @@ def heatmaps_to_keypoints(self, heatmaps, threshold=HM_TO_KP_THRESHOLD):
             # and get its coordinates and confidence
             y, x = np.where(peaks == peaks.max())
             if int(x[0]) > 0 and int(y[0]) > 0:
```

Comment:
Unrelated to your changes. Do we need to cast to int here? This would turn a 0.99 into 0. Is this what we want? @nicolepeverley

Reply:
I think I just left the int cast from the original code in the other repo (https://github.com/yuanyuanli85/Stacked_Hourglass_Network_Keras/blob/eddf0ae15715a88d7859847cfff5f5092b260ae1/src/eval/heatmap_process.py#L5). I'm pretty sure they should all already be ints. I just tested a couple images and they all appear to be 0 or another int within 255.